cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -63,25 +63,37 @@ func runCleanupNode(c workflow.RunData) error {
 		klog.Warningln("[reset] The kubelet service could not be stopped by kubeadm. Unable to detect a supported init system!")
 		klog.Warningln("[reset] Please ensure kubelet is stopped manually")
 	} else {
-		fmt.Println("[reset] Stopping the kubelet service")
-		if err := initSystem.ServiceStop("kubelet"); err != nil {
-			klog.Warningf("[reset] The kubelet service could not be stopped by kubeadm: [%v]\n", err)
-			klog.Warningln("[reset] Please ensure kubelet is stopped manually")
+		if !r.DryRun() {
+			fmt.Println("[reset] Stopping the kubelet service")
+			if err := initSystem.ServiceStop("kubelet"); err != nil {
+				klog.Warningf("[reset] The kubelet service could not be stopped by kubeadm: [%v]\n", err)
+				klog.Warningln("[reset] Please ensure kubelet is stopped manually")
+			}
+		} else {
+			fmt.Println("[reset] Would stop the kubelet service")
```

Comment:
just wanted to note that, i was trying to figure out if we are consistent in how we print dryrun actions. do we prefix with the command name e.g. [reset] or do we always prefix with [dryrun].

it seems we do both and it's not very consistent.


<details> 

```
grep "\[dryrun\]" * -rnI

app/phases/controlplane/manifests.go:106:			fmt.Printf("[dryrun] Would create users and groups for %+v to run as non-root\n", componentNames)
app/phases/controlplane/manifests.go:130:				fmt.Printf("[dryrun] Would update static pod manifest for %q to run run as non-root\n", componentName)
app/phases/upgrade/health.go:107:		fmt.Printf("[dryrun] Would create the Job %q in namespace %q and wait until it completes\n", jobName, ns)
app/phases/upgrade/postupgrade.go:254:		fmt.Printf("[dryrun] Would ensure that %q includes a CRI endpoint URL scheme\n", filePath)
app/phases/etcd/local.go:149:		fmt.Printf("[dryrun] Would add etcd member: %s\n", etcdPeerAddress)
app/phases/etcd/local.go:167:		fmt.Println("[dryrun] Would wait for the new etcd member to join the cluster")
app/phases/etcd/local.go:267:			fmt.Printf("[dryrun] Would create users and groups for %q to run as non-root\n", kubeadmconstants.Etcd)
app/phases/etcd/local.go:268:			fmt.Printf("[dryrun] Would update static pod manifest for %q to run run as non-root\n", kubeadmconstants.Etcd)
app/util/apiclient/dryrunclient.go:128:						fmt.Println("[dryrun] Returning faked GET response:")
app/util/apiclient/dryrunclient.go:153:						fmt.Println("[dryrun] Returning faked LIST response:")
app/util/apiclient/dryrunclient.go:215:	fmt.Fprintf(w, "[dryrun] Would perform action %s on resource %q in API group \"%s/%s\"\n", strings.ToUpper(action.GetVerb()), action.GetResource().Resource, group, action.GetResource().Version)
app/util/apiclient/dryrunclient.go:219:		fmt.Fprintf(w, "[dryrun] Resource name: %q\n", namedAction.GetName())
app/util/apiclient/dryrunclient.go:227:			fmt.Println("[dryrun] Attached object:")
app/util/apiclient/dryrunclient.go:235:		fmt.Fprintf(w, "[dryrun] Attached patch:\n\t%s\n", strings.Replace(string(patchAction.GetPatch()), `\"`, `"`, -1))
app/util/apiclient/dryrunclient_test.go:40:			expectedBytes: []byte(`[dryrun] Would perform action GET on resource "services" in API group "core/v1"
app/util/apiclient/dryrunclient_test.go:41:[dryrun] Resource name: "kubernetes"
app/util/apiclient/dryrunclient_test.go:47:			expectedBytes: []byte(`[dryrun] Would perform action GET on resource "clusterrolebindings" in API group "rbac.authorization.k8s.io/v1"
app/util/apiclient/dryrunclient_test.go:48:[dryrun] Resource name: "system:node"
app/util/apiclient/dryrunclient_test.go:54:			expectedBytes: []byte(`[dryrun] Would perform action LIST on resource "services" in API group "core/v1"
app/util/apiclient/dryrunclient_test.go:67:			expectedBytes: []byte(`[dryrun] Would perform action CREATE on resource "services" in API group "core/v1"
app/util/apiclient/dryrunclient_test.go:82:			expectedBytes: []byte(`[dryrun] Would perform action PATCH on resource "nodes" in API group "core/v1"
app/util/apiclient/dryrunclient_test.go:83:[dryrun] Resource name: "my-node"
app/util/apiclient/dryrunclient_test.go:84:[dryrun] Attached patch:
app/util/apiclient/dryrunclient_test.go:91:			expectedBytes: []byte(`[dryrun] Would perform action DELETE on resource "pods" in API group "core/v1"
app/util/apiclient/dryrunclient_test.go:92:[dryrun] Resource name: "my-pod"
app/util/apiclient/clientbacked_dryrun.go:132:		fmt.Println("[dryrun] The GET request didn't yield any result, the API Server returned a NotFound error.")
app/util/dryrun/dryrun.go:78:		fmt.Fprintf(w, "[dryrun] Would write file %q with content:\n", outputFilePath)
app/util/dryrun/dryrun.go:94:	fmt.Println("[dryrun] Would wait for the API Server's /healthz endpoint to return 'ok'")
app/util/dryrun/dryrun.go:100:	fmt.Printf("[dryrun] Would wait for the Pods with the label %q in the %s namespace to become Running\n", kvLabel, metav1.NamespaceSystem)
app/util/dryrun/dryrun.go:106:	fmt.Printf("[dryrun] Would wait for the %q Pod in the %s namespace to be deleted\n", podName, metav1.NamespaceSystem)
app/util/dryrun/dryrun.go:112:	fmt.Printf("[dryrun] Would make sure the kubelet %q endpoint is healthy\n", healthzEndpoint)
app/util/dryrun/dryrun.go:149:		fmt.Printf("[dryrun] Wrote certificates, kubeconfig files and control plane manifests to the %q directory\n", manifestDir)
app/util/dryrun/dryrun.go:156:		fmt.Printf("[dryrun] Wrote certificates and kubeconfig files to the %q directory\n", manifestDir)
app/util/dryrun/dryrun.go:159:	fmt.Println("[dryrun] The certificates or kubeconfig files would not be printed due to their sensitive nature")
app/util/dryrun/dryrun.go:160:	fmt.Printf("[dryrun] Please examine the %q directory for details about what would be written\n", manifestDir)
app/cmd/phases/join/controlplanejoin.go:139:		fmt.Printf("[dryrun] Would ensure that %q directory is present\n", cfg.Etcd.Local.DataDir)
app/cmd/phases/join/controlplanejoin.go:197:		fmt.Printf("[dryrun] Would mark node %s as a control-plane\n", cfg.NodeRegistration.Name)
app/cmd/phases/join/kubelet.go:173:		fmt.Println("[dryrun] Would stop the kubelet")
app/cmd/phases/join/kubelet.go:190:		fmt.Println("[dryrun] Would start the kubelet")
app/cmd/phases/upgrade/node/kubeletconfig.go:121:				fmt.Println("[dryrun] would update the node CRI socket path to include an URL scheme")
app/cmd/phases/init/etcd.go:96:				fmt.Printf("[dryrun] Would ensure that %q directory is present\n", cfg.Etcd.Local.DataDir)
app/cmd/upgrade/apply.go:173:		fmt.Println("[dryrun] Finished dryrunning successfully!")


```

</details> 

but overall that does not seem like a big issue, i think we should keep only global dryrun utilities prefixed with [dryrun]
such an utility is `app/util/apiclient/dryrunclient.go` and not sure what else qualifies.

i think having `[reset] Would ...` for this PR is fine.